src/components/SpatialProfiler/SpatialProfilerSettingsPanelComponent/SpatialProfilerSettingsPanelComponent.tsx

```diff
@@ -1,6 +1,7 @@
 import * as React from "react";
 import {computed, autorun} from "mobx";
 import {observer} from "mobx-react";
+import {CARTA} from "carta-protobuf";
```

Comment:
`carta-protobuf` import should go below all third-party imports

Reply:
I am using Visual Studio Code. Around 20 thousands line were deleted from the package-lock.json when I npm install initially. I corrected the file to match the one in the dev branch. I mistakenly used "git add ." to stage the changed files. In the future, I will use "git add 'filename'" to stage only the nessesary files in order to avoid confusion.